### PR TITLE
feat(suite-native): show consistent Tokens label across the app

### LINKED ISSUE #14209: Coins with tokens label change
Whenever we mention coin which includes tokens we support and present in the app we should be consistent in their appearance within the app.
Acceptance criteria:
1. When user is prompted to select coins he would like to use with the app (coin enabling) and there are coins including tokens, the label say [coin] +tokens
2. When user goes to Enabled coins settings and there are coins including tokens, the label say [coin] +tokens
3. When user wants to sync coin in portfolio tracker and he selects the coin and there are coins including tokens, the label say [coin] +tokens

Figma: https://www.figma.com/design/oQtgXoPIgGSlh01TLHnmeE/Coin-enabling?node-id=381-25514&node-type=FRAME&t=qIbNfHggugBL7yx9-0

### PR DESCRIPTION
**Pull request by vytick**
Make visual and appearance of badge ```+ Tokens``` consistent across the app

Resolve #14209

## Screenshots:

https://github.com/user-attachments/assets/494e662e-ef9b-49e5-9f5c-cda38c9778c2

### CODE REVIEW

File: suite-native/intl/src/en.ts [hunk 1]
```diff
@@ -84,7 +85,7 @@ export const en = {
         },
     },
     accountList: {
-        numberOfTokens: '+{numberOfTokens} tokens',
+        numberOfTokens: '+{numberOfTokens} Tokens',
```
Comment (yanascz) [inline suite-native/intl/src/en.ts:88]: How is plurarization handled? Does it say `+1 Token` or `+1 Tokens`?

Comment (vytick) [reply suite-native/intl/src/en.ts:88]: I expected this to be handled by the task that should resolve current default +1. Added here just not to forget to capitalise the first letter to make it consistent.

Do you think it should be handled here?

File: suite-native/accounts/src/components/SelectableNetworkItem.tsx [hunk 2]
```diff
@@ -35,7 +37,7 @@ export const SelectableNetworkItem = ({ symbol, onPress, rightIcon }: Selectable
 
     const networkName = networks[symbol].name;
 
-    const isEthereumNetwork = symbol === 'eth';
+    const hasWithTokens = isCoinWithTokens(symbol);
```
Comment (yanascz) [inline suite-native/accounts/src/components/SelectableNetworkItem.tsx]: This reads nicer.
```suggestion
    const hasTokens = isCoinWithTokens(symbol);
```

File: suite-native/accounts/src/components/SelectableNetworkItem.tsx [hunk 3]
```diff
@@ -55,9 +57,13 @@ export const SelectableNetworkItem = ({ symbol, onPress, rightIcon }: Selectable
                                     areAmountUnitsEnabled={false}
                                 />
                             </Text>
-                            {isEthereumNetwork && (
+                            {hasWithTokens && (
                                 <Box style={applyStyle(erc20BadgeStyle)}>
```
Comment (yanascz) [inline suite-native/accounts/src/components/SelectableNetworkItem.tsx]: The style naming seems obsolete. `tokenBadgeStyle` maybe?

File: suite-native/intl/src/en.ts (hunk 1 above)
Comment (yanascz) [reply suite-native/intl/src/en.ts:88]: OK. I don't mind, as long as it's handled somewhere. 🙂

**Review by yanascz — APPROVED** [review]
Looks good! Added some suggestions for tiny improvements, though.

**Review by Nodonisko — APPROVED** [review]

File: suite-native/accounts/src/components/SelectableNetworkItem.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/accounts/src/components/SelectableNetworkItem.tsx]: 8d0a642c36

File: suite-native/accounts/src/components/SelectableNetworkItem.tsx (hunk 3 above)
Comment (vytick) [reply suite-native/accounts/src/components/SelectableNetworkItem.tsx]: 8d0a642c36

**Pull request merged**